src/main/java/de/zalando/ep/zalenium/container/kubernetes/KubernetesContainerClient.java

```diff
@@ -109,7 +109,7 @@ public KubernetesContainerClient(Environment environment,
                             + "\tSelenium Pod Resource Limits: %s\n"
                             + "\tSelenium Pod Resource Requests: %s",
                     hostname, appName, zaleniumAppName,
-                            seleniumPodLimits.toString(), seleniumPodRequests.toString()));
```

Comment:
Please revert changes to this file, it is just some spaces and tabs that changed.